inventory-web-app/tailwind.config.js

```diff
@@ -0,0 +1,14 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: [
```

Comment:
If we're going to use tailwind please configure to purge unused css on build to avoid large bundle sizes on production
https://v2.tailwindcss.com/docs/optimizing-for-production